plugins/admin/pkg/command/domain/set.go

```diff
@@ -19,22 +19,28 @@ import (
 	"fmt"
 	"strings"
 
+	"knative.dev/client-contrib/plugins/admin/pkg/command/utils"
+
 	"knative.dev/client-contrib/plugins/admin/pkg"
 
 	"github.com/spf13/cobra"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/equality"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var selector []string
-var domain string
+var (
+	ScaleToZero    bool
```

Comment:
Why is `ScaleToZero` var public when the others are private?

Reply:
Thanks Max! this is typo and fixed.